### ISSUE TITLE
Default configurations are not found.

### ISSUE THREAD

**Issue by petsagouris** [DeAlexPesh/totum-docker-compose#2]
https://github.com/DeAlexPesh/totum-docker-compose/blob/ba7e2bd81f8b32621875228100109115dc4f3168/images/totum/Dockerfile#L91

These COPY directives need the files inside the `/conf`directory. I don't see why they're not moved.

**Comment by DeAlexPesh** [comment]
![image](https://github.com/DeAlexPesh/totum-docker-compose/assets/16735261/205ddff6-73a7-4dec-a55b-2cd67e1a92d3)

Files exist near dockerfile?

readme?
![image](https://github.com/DeAlexPesh/totum-docker-compose/assets/16735261/ccb6758c-2604-4c66-ae97-6f3a30d4b840)

**Comment by petsagouris** [comment]
So following the readme goes like this on my machine:
```
PS C:\dev> git clone https://github.com/DeAlexPesh/totum-docker-compose.git app
Cloning into 'app'...
remote: Enumerating objects: 33, done.
remote: Counting objects: 100% (33/33), done.
remote: Compressing objects: 100% (24/24), done.
remote: Total 33 (delta 3), reused 0 (delta 0), pack-reused 0
Receiving objects: 100% (33/33), 8.88 KiB | 2.96 MiB/s, done.
Resolving deltas: 100% (3/3), done.
PS C:\dev> cd app
PS C:\dev\app> docker build -t images/totum:mode  /app/images/totum/. -f /app/images/totum/Dockerfile
[+] Building 0.6s (16/17)                                                                                docker:default
 => [internal] load build definition from Dockerfile                                                               0.0s
 => => transferring dockerfile: 3.05kB                                                                             0.0s
 => [internal] load .dockerignore                                                                                  0.0s
 => => transferring context: 2B                                                                                    0.0s
 => [internal] load metadata for docker.io/library/ubuntu:focal                                                    0.0s
 => [ 1/13] FROM docker.io/library/ubuntu:focal                                                                    0.0s
 => [internal] load build context                                                                                  0.0s
 => => transferring context: 85B                                                                                   0.0s
 => CANCELED [ 2/13] RUN apt-get -q update  && apt-get install -qy --no-install-recommends     apt-utils     loca  0.5s
 => CACHED [10/13] COPY [./conf/supervisord.conf, /etc/supervisor/conf.d/totum.conf]                               0.0s
 => ERROR [ 8/13] COPY [./conf/totum-nginx.conf, /etc/nginx/sites-available/default]                               0.0s
 => CACHED [ 4/13] RUN git clone https://github.com/totumonline/totum-mit.git totum                                0.0s
 => CACHED [ 5/13] WORKDIR /home/totum/totum                                                                       0.0s
 => CACHED [ 6/13] RUN php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"  && php composer  0.0s
 => ERROR [ 7/13] COPY [./conf/totum-php.conf, /etc/php/8.0/fpm/pool.d/totum.conf]                                 0.0s
 => ERROR [ 9/13] COPY [./conf/totum.cronjob, /etc/cron.d/totum]                                                   0.0s
 => CACHED [11/13] COPY [./conf/fls/, /home/totum/totum/http/fls/]                                                 0.0s
 => ERROR [12/13] COPY [./conf/entrypoint.sh, /entrypoint.sh]                                                      0.0s
 => CACHED [ 3/13] WORKDIR /home/totum                                                                             0.0s
------
 > [ 8/13] COPY [./conf/totum-nginx.conf, /etc/nginx/sites-available/default]:
------
------
 > [ 7/13] COPY [./conf/totum-php.conf, /etc/php/8.0/fpm/pool.d/totum.conf]:
------
------
 > [ 9/13] COPY [./conf/totum.cronjob, /etc/cron.d/totum]:
------
------
 > [12/13] COPY [./conf/entrypoint.sh, /entrypoint.sh]:
------
Dockerfile:96
--------------------
  94 |     COPY ["./conf/supervisord.conf", "/etc/supervisor/conf.d/totum.conf"]
  95 |     COPY ["./conf/fls/", "/home/totum/totum/http/fls/"]
  96 | >>> COPY ["./conf/entrypoint.sh", "/entrypoint.sh"]
  97 |
  98 |     RUN ln -s /home/totum/totum/bin/totum /bin/totum \
--------------------
ERROR: failed to solve: failed to compute cache key: failed to calculate checksum of ref 52133a48-d4eb-4050-a3fb-1dca15a2e57e::v9mf1u5evre8ukoxidx87hyv9: "/conf/entrypoint.sh": not found

View build details: docker-desktop://dashboard/build/default/default/aua4jd0b9vt1xnjrcmolh6b4t
PS C:\dev\app>
```

**Comment by DeAlexPesh** [comment]
`PS C:\dev> cd app` - it is not necessary to do this

on windows like os `./` this often causes an error, replace  by absolut path and all is done work.
